docs/reference/components.cue

```diff
@@ -776,9 +776,51 @@ components: {
 
 			_types: {
 				common:      true
-				description: "Key/value pairs representing mapped log field names and types. This is used to coerce log fields into their proper types."
+				description: """
+					Key/value pairs representing mapped log field names and types. This is used to
+					coerce log fields from strings into their proper types. The available types are
+					listed in the **Types** list below.
+
+					Timestamp coercions need to be prefaced with `timestamp|`, for example
+					`\"timestamp|%F\"`. Timestamp specifiers can use either of the following:
+
+					1. One of the built-in-formats listed in the **Timestamp Formats** table below.
+					2. The [time format specifiers](\(urls.chrono_time_formats)) from Rust's
```

Comment:
I think it might be preferable to copy the supported specifiers into here rather than making users look at documentation for a Rust library.

Reply:
I don't disagree in principle but from a practical perspective there are two drawbacks:

1. We'd be on the hook to keep that information up to date on our side, and the amount of information is non trivial.
1. We'd be displaying three pretty sizable tables in the `type` section. You can see an example of such a section in the [`grok_parser`](https://vector.dev/docs/reference/transforms/grok_parser/#types) docs. Those sections are supposed to be simple and bite sized, and including even these two tables already feels like pushing it.

Reply:
That's fair. I just feel like it seems a little rough to point users to development documentation. Still, this is an improvement :+1: